rust_src/src/buffers.rs

```diff
@@ -320,6 +335,57 @@ pub fn overlay_buffer(overlay: LispObject) -> LispObject {
     marker_buffer(marker)
 }
 
+pub fn recenter_overlay_lists(mut buf: LispBufferRef, pos: ptrdiff_t) {
+    // let mut overlay: LispObject;
```

Comment:
i don't know that creating these 3 variables at once is cleaner than 3 let statements.